public/themes/welcomeTheme.js

```diff
@@ -0,0 +1,14 @@
+import { createTheme } from '@mui/material/styles';
```

Comment:
let's use default.js as the name of the file and theme. Or even better salesWell.js

Reply:
Also, are we sure we need to create a theme?